sqlserver/datadog_checks/sqlserver/data/conf.yaml.example

```diff
@@ -57,7 +57,7 @@ instances:
     # database: master
 
     ## @param database_autodiscovery - boolean - optional - default: false
-    ## Auto-discover and monitor databases.  If `true`, overrides `database` option.
+    ## Auto-discover and monitor databases. This param is supported for the metrics check.  If `true`, overrides `database` option.
```

Comment:
The description needs to be updated here as well: https://github.com/DataDog/integrations-core/blob/b28f571404f73092c59b03336d3b573c2d1a8c48/sqlserver/assets/configuration/spec.yaml

Reply:
good catch. Updated!